src/store.js

```diff
@@ -90,7 +90,10 @@ const useStoreImpl = create((set, get) => {
       right: false,
       brake: false,
       honk: false,
-      boost: false,
+    },
```

Comment:
not sure if we want boost outside of controls like this.. perhaps someone else can weigh in

Reply:
ye me don't like it as well outside, but the 3 deep nesting of the object props is also ugly :(

Reply:
If has nobody more concerns or objections regarding boost object nesting or other reasons, then I will merge this pr in 30 mins. Please rise your hands :)

Reply:
✋🏽